packages/pwa-kit-dev/src/configs/eslint/eslint-config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     plugins: ['header', 'react', 'prettier'],
     settings: {
         react: {
-            version: '16.8'
+            version: 'detect'
```

Comment:
While bumping the dependency I noticed that this property was pointing to an old React version (since the project is using v17 currently). To avoid having to manually sync this, we can use the `detect` option.

Everything appears to be working as expected, the only difference is with the `template-express-minimal` package where you get the following warning:

> template-express-minimal: Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.

However I wouldn't say it's an issue, since as far as I understand that package isn't intended for usage with React in the first place (and if React _was_ installed, ESLint would pick up the correct version).

Reply:
TIL about the `detect` option, nice catch.

You're right, the `template-express-minimal` is not intended for usage with React and the warning is not a problem.